subprojects/docs/src/docs/userguide/build_cache.adoc

```diff
@@ -43,31 +43,31 @@ Put `org.gradle.caching=true` in your `gradle.properties`::
 Gradle will try to reuse outputs from previous builds for all builds, unless explicitly disabled with `--no-build-cache`.
 
 When the build cache is enabled, it will store build outputs in the Gradle user home.
-For configuring this directory or different kinds of build caches see <<sec:build_cache_configure>>.
+For configuring this directory or different kinds of build caches see <<#sec:build_cache_configure,Configure the Build Cache>>.
 
 [[sec:task_output_caching]]
-=== Task Output Caching
+== Task Output Caching
 
-Beyond incremental builds described in <<sec:up_to_date_checks>>, Gradle can save time by reusing outputs from previous executions of a task by matching inputs to the task.
+Beyond incremental builds described in <<more_about_tasks.adoc#sec:up_to_date_checks,up-to-date checks>>, Gradle can save time by reusing outputs from previous executions of a task by matching inputs to the task.
 Task outputs can be reused between builds on one computer or even between builds running on different computers via a build cache.
 
 We have focused on the use case where users have an organization-wide remote build cache that is populated regularly by continuous integration builds.
 Developers and other continuous integration agents should load cache entries from the remote build cache.
 We expect that developers will not be allowed to populate the remote build cache, and all continuous integration builds populate the build cache after running the `clean` task.
 
-For your build to play well with task output caching it must work well with the <<sec:up_to_date_checks,incremental build>> feature.
+For your build to play well with task output caching it must work well with the <<more_about_tasks.adoc#sec:up_to_date_checks,incremental build>> feature.
 For example, when running your build twice in a row all tasks with outputs should be `UP-TO-DATE`.
 You cannot expect faster builds or correct builds when enabling task output caching when this prerequisite is not met.
 
-Task output caching is automatically enabled when you enable the build cache, see <<sec:build_cache_enable>>.
+Task output caching is automatically enabled when you enable the build cache, see <<#sec:build_cache_enable,Enable the Build Cache>>.
 
 [[sec:task_output_caching_example]]
-==== What does it look like
+=== What does it look like
 
 Let us start with a project using the Java plugin which has a few Java source files. We run the build the first time.
 
 ----
-$> gradle --build-cache compileJava
+> gradle --build-cache compileJava
```

Comment:
Did you remove the `$` signs for consistency with other chapters?

Reply:
Yes

Reply:
👍